daprdocs/content/en/dotnet-sdk-docs/_index.md

```diff
@@ -32,5 +32,4 @@ git clone https://github.com/dapr/dotnet-sdk.git
 
 ## More information
 
-- [NuGet packages](https://www.nuget.org/profiles/dapr.io)
-- [Dapr SDK serialization]({{< ref sdk-serialization.md >}})
```

Comment:
I'm not sure what this is a reference to so I removed it.

Reply:
I think this refers to this page: https://docs.dapr.io/developing-applications/sdks/sdk-serialization/ 
Right now it has details on serialization in Java.

Also does it make sense to add the Understanding serialization section for .NET also to this page?

Reply:
I think that would be counterproductive to combine them all if they are all different. If I a .NET user I'd come here first 😆